openmc/deplete/operator.py

```diff
@@ -267,8 +266,8 @@ def __call__(self, vec, power):
         ----------
         vec : list of numpy.ndarray
             Total atoms to be used in function.
-        power : float
-            Power of the reactor in [W]
+        source_rate : float
+            Source rate in [W] in [neutron/sec]
```

Comment:
```suggestion
            Source rate in [W] or in [neutron/sec]
```

Reply:
Maybe even better to mach the doc string from `TransportOperator`:

"Power in [W] or source rate in [neutron/sec]"